pyproject.toml

```diff
@@ -79,7 +79,11 @@ dev-mkdocs = [
 ]
 dev-mypy = [
   "mypy == 1.5.1",
-  "types-setuptools >= 67.6.0, < 68", # Should match the global dependency
+  "types-setuptools == 68.1.0.0", # Should match the build dependency
+  "types-Markdown == 3.4.2.10",
```

Comment:
How is this updated when the mypy version changes? Like, who keeps it in sync? Dependabot would just pick the latest versions no? And if those don't match what mypy deps need?